docs/application-connector/03-08-application-gateway-proxying-requests.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Proxying requests by the Application Gateway
+type: Details
+---
+
+The Application Gateway proxies requests from lambda functions and services in Kyma to external APIs. When the Application Gateway works in the alternative `gatewayOncePerNamespace` [mode](#architecture-application-connector-components-application-operator), it proxies the requests based on the configuration stored in Secrets.
+​
+### Proxy configuration
+​
+The Secret expected by the Application Gateway has the following structure:
+
+```
+data:
+    {API_NAME_1}_TARGET_URL: {BASE64_ENCODED_API1_URL}
+    {API_NAME_2}_TARGET_URL: {BASE64_ENCODED_API2_URL}
+    CONFIGURATION: {BASE64_ENCODED_CONFIG_JSON}
+    CREDENTIALS_TYPE: {BASE64_ENCODED_CREDENTIALS_TYPE}
+```
+
+* The `BASE64_ENCODED_CONFIG_JSON` configuration contains credentials and request parameters. 
+* The `BASE64_ENCODED_CREDENTIALS_TYPE` assumes one of the following values:  `OAuth`, `BasicAuth`, `Certificate` (not supported in the Director), `NoAuth`.
+​
+
+An example `CONFIGURATION` for APIs secured with OAuth looks as follows:
```

Comment:
```suggestion
The example `CONFIGURATION` for APIs secured with OAuth looks as follows:
```

Reply:
Also: do we need examples for other authorization types?

Reply:
I think it should be "an", but the other one is a good question. I'll check.

Reply:
Yeah, I'll add an example for the `BasicAuth` method, too. Thanks.